File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        int sum=0;
        for(auto x:s)
        {
            int digit=x-'a'+1;
            sum+=(digit/10)+(digit%10);

        }
        while(k>1)
        {
            sum=digitsum(sum);
            k--;
        }
        return sum;
    }

        int digitsum(int num)
        {
            int sum = 0;
            while (num > 0)
             {
                sum += num % 10;
                num /= 10;
        }
            return sum;
        }
};

File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.py
class Solution(object):
    def getLucky(self, s, k):
        """
        :type s: str
        :type k: int
        :rtype: int
        """
        num = 0
        total = 0
        for c in s:
            num = ord(c) - 96
            while num > 0:
                total += num % 10
                num = num // 10
        num = total
        total = 0

        for _ in range(k-1):
            while num > 0:
                total += num % 10
                num = num // 10
            num = total
            total = 0
        return num